src/main/kotlin/dynamic_programming/PalindromePartitioning.kt

```diff
@@ -0,0 +1,75 @@
+package dynamic_programming
+
+/**
+ * Palindrome Partitioning Algorithm
+ *
+ * You are given a string as input, and task is to find the minimum number of partitions to be made,
+ * in the string sot that the resulting strings are all palindrome
+ *
+ * Time Complexity = O(n^2)
+ *
+ * */
+
+
+/**
+ * @param String is the string to be checked
```

Comment:
Add 1-2 line description for this function

Reply:
I believe the name isPalindrome is self-explaining enough, writing a description would be over-explaining the obvious